src/main/java/booktopiahub/repository/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package booktopiahub.repository;
+
+import booktopiahub.model.Book;
+import java.util.List;
+
+public interface BookRepository {
```

Comment:
Use the JpaRepository
https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa